### DISCUSSION TITLE
level=info msg=Encountered self-service settings error. audience=audit error=map[message:I[#/traits] S[#/properties/traits/additionalProperties] additionalProperties "avatar" not allowed]

### DISCUSSION THREAD

**Discussion by flaviobzs** [ory/kratos#1161] [Q&A] [answered]
time=2021-03-18T15:22:44Z level=info msg=Encountered self-service settings error. audience=audit error=map[message:I[#/traits] S[#/properties/traits/additionalProperties] additionalProperties "avatar" not allowed] http_request=map[headers:map[accept:text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9 accept-encoding:gzip, deflate, br accept-language:en-US,en;q=0.9,pt;q=0.8 cache-control:max-age=0 cookie:csrf_token=ONc+S2gBq220Qd48ub4vedCaFcfxoL88ZooFOo9FpNI=; csrf_token=8HtRImO6i3xo6RLlRbb8JTtV5NlmwQDVTaTm6HxlsN4=; ory_kratos_session=MTYxNjA4MDg3MnxEdi1CQkFFQ180SUFBUkFCRUFBQVJfLUNBQUVHYzNSeWFXNW5EQThBRFhObGMzTnBiMjVmZEc5clpXNEdjM1J5YVc1bkRDSUFJRzQzYWpKNlJXNWFjbWxCWkRCM2FtRlpjRXd4Vld0WlRrZFhWWEJaTUdOSXyMH8BQ4W_pX8NmelVnD5qZLIXcsKLqFTBMgVAi4mA5vQ== origin:http://127.0.0.1:4455 referer:http://127.0.0.1:4455/auth/settings?flow=8701108c-2d31-4e5f-b880-4d4a9fcbb09d user-agent:Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36 x-forwarded-for:192.168.96.1] host:127.0.0.1:4455 method:POST path:/self-service/settings/methods/password query:flow=8701108c-2d31-4e5f-b880-4d4a9fcbb09d remote:192.168.96.4:46092 scheme:http] service_name=ORY Kratos service_version=v0.5.5-alpha.1.pre.1 settings_flow=&{8701108c-2d31-4e5f-b880-4d4a9fcbb09d browser 2021-03-18 16:21:12.469656 +0000 UTC 2021-03-18 15:21:12.469656 +0000 UTC http://127.0.0.1:4455/self-service/recovery/methods/link?token=U6tGNfsmEwzC1NtzNrC9PlID1VeDbad3  [{1060001 You successfully recovered your account. Please change your password or set up an alternative login method (e.g. social sign in) within the next 15.00 minutes. info [123 34 112 114 105 118 105 108 101 103 101 100 83 101 115 115 105 111 110 69 120 112 105 114 101 115 65 116 34 58 32 34 50 48 50 49 45 48 51 45 49 56 84 49 53 58 51 54 58 49 50 46 52 55 54 53 55 52 53 90 34 125]}] map[password:0xc0002f4b60 profile:0xc0002f4af0] [] 0xc000e669c0 show_form 250b5c12-4f47-42fa-8171-a9a3283088e7 2021-03-18 15:21:12.47016 +0000 UTC 2021-03-18 15:21:12.47939 +0000 UTC}

**Comment by flaviobzs** [comment, marked as answer]
I'm trying to implement a password reset, but, I always follow this error, even generating the form with only the fields indicated in the settings flow response

**Reply by Benehiko** [reply]
Hi @flaviobzs,

Thank you for reaching out :smiley: 

Could you please provide me with your **kratos configuration.yml** file and **identity.json** file as well as your **json payloads** you are sending to the endpoint giving you the issues.
 
Also please format your code/json/console log properly, see [here](https://guides.github.com/features/mastering-markdown/#examples) how to do that.

**Reply by vinckr** [reply]
Hey , just checking in if your problem still persists @flaviobzs . 
I will mark it as answered for now.
Thanks a ton 🙏
